straxbra/plugins.py

```diff
@@ -321,68 +321,30 @@ class PeakPositions(strax.Plugin):
     parallel = True
 
     def setup(self):
-        # PMT mask
-        self.pmt_mask = np.zeros_like(self.config['to_pe'], dtype=np.bool)
-        self.pmt_mask[self.config['top_pmts']] = self.config['to_pe'][self.config['top_pmts']] > 0
-        # PMT positions in mm in cartesian coordinates
-        pmt_x = np.array([-14.,-28.,-14.,14.,28.,14.,0.]) # x-positions PMTs
-        pmt_x = pmt_x[self.pmt_mask[self.config['top_pmts']]]
-        pmt_y = np.array([-28.,0.,28.,28.,0.,-28.,0.])    # y-positions PMTs
-        pmt_y = pmt_y[self.pmt_mask[self.config['top_pmts']]]
-        self.pmt_positions = np.column_stack((pmt_x, pmt_y))
-        # Fit parameters LRFs
-        # MC driven (R_PTFE = 95 %, T_meshes = 89.770509 %, lambda_LXe = 100 cm); 
-        # To be iteratively determined from data later
-        self.fitparameters = np.array([[ 0.58534179, 29.89341846, -0.3275816, 4.14715081, 2.61234684], 
-                          [ 0.63546614, 28.49525342, -0.19651583, 3.76011493, 2.68672152], 
-                          [ 0.58840586, 30.39015033, -0.38248759, 4.27642323, 2.56774842], 
-                          [ 0.59111988, 31.38350968, -0.5760228, 4.64122418, 2.51311592], 
-                          [ 0.63771524, 28.90663204, -0.26194541, 3.90052756, 2.66510948], 
-                          [ 0.59030322, 30.55082687, -0.46183924, 4.39769959, 2.55446814], 
-                          [ 0.53114467, 39.20861977, -17.93187819, 20.60397171, 2.27692367]])
-        self.fitparameters = self.fitparameters[self.pmt_mask[self.config['top_pmts']]]
+        ## Load the trained model from corresponding HDF5 file
+        self.model_NN = keras.models.load_model('XeBRA_Position_Reconstruction_NN_Model.h5')
```

Comment:
Is this file in the folder where the script is running? Is it in the folder where the script is installed? Add the full path. Also, perhaps consider adding something to the filename indicating that it's for the dual-phase 7-PMT configuration.

Reply:
The indication of the dual-phase 7-PMT configuration is added. The HDF5 file is currently in the same folder as plugins.py, therefore no path indication. What full path would you recommend to add resp. where is the file most conveniently stored?

Reply:
Since most of the directories appear to have restricted access, I moved it to `/data/workspace/nn_models`.